design/ceph/object/store.md

```diff
@@ -73,7 +75,53 @@ If `preservePoolsOnDelete` is set to 'true' the pools used to support the object
   preservePoolsOnDelete: true
 ```
 
-If there is a `zone` section in object-store configuration, then the pool section in the ceph-object-zone resource will be used to define the pools.
+#### Pools shared by multiple CephObjectStore
+
+If user want to use existing pools for metadata and data, the pools must be created before the object store is created. This will be useful if multiple objectstore can share same pools. The detail of pools need to shared in `radosNamespaces` settings in object-store CRD. And for this option to work, `preservePoolsOnDelete` must be set to 'true'. Now the object stores can consume same pool isolated with different namespaces. User can create via [Pool CRD](/Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md), this is need to present before the object store is created.
+
+```yaml
+  radosNamespaces:
+    metadataPoolName: rgw-meta-pool
+    dataPoolName: rgw-data-pool
+  preservePoolsOnDelete: true
+```
+
+And the pool CRD will look like this:
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+    name: rgw-meta-pool
+  spec:
+    failureDomain: host
+    replicated:
+      size: 3
+    parameters:
+      pg_num: 8
+    application: rgw
+---
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+ metadata:
+    name: rgw-data-pool
+  spec:
+      failureDomain: osd
+      erasureCoded:
+        dataChunks: 6
+        codingChunks: 2
+    application: rgw
+```
+
+#### For mulisite use case
+
+If there is a `zone` section in object-store configuration, then the pool creation will performed by the [CephObjectZone CRD](/Documentation/CRDs/Object-Storage/ceph-object-zone-crd.md). The `CephObjectStore` CRD will include below section to specify the zone name.
```

Comment:
I'm not sure I understand. Does this mean zones will always have different pools? Can we allow zones to use namespaces as well?

Reply:
Yes, for zones pool config is part of CephObjectZone CRD and created along with it, see https://rook.io/docs/rook/v1.12/CRDs/Object-Storage/ceph-object-zone-crd/ for details.
This option is applicable for normal object store in which pool creation happens with CephObjectStore CRD

Reply:
I'm not clear either. So RGW multisite cannot use rados namespaces in the pools?

Reply:
The option can be applicable to normal object stores while the pool is configured within in objectstore CRD. The multisite pool settings are configured with Zone settings. The feature more inclined with normal objectstore, we can support it for multisite. What I meant was for multisite multiple object stores already sharing the pools created by the zone.